site/gatsby-site/blog/incident-report-2022-october/index.fr.mdx

```diff
@@ -1,143 +1,142 @@
+* **Que s'est-il passé ?** Le modèle de modération des utilisateurs du site de médias sociaux chinois Weibo a du mal à suivre l'évolution de l'argot des utilisateurs au mépris des censeurs de l'État chinois.
+* **Comment l'IA _ne_ fonctionne-t-elle pas ?** Bien que le site affirme avoir affiné son "modèle d'identification des mots clés" pour pouvoir filtrer l'utilisation de mots intentionnellement mal orthographiés et d'homophones, la diversité et la nature en constante évolution du langage en ligne en Chine rend peu probable que son modèle soit en mesure d'interdire totalement le langage controversé.
+* **Quel a été l'impact de cet incident ?** Les citoyens chinois ont pu saper les efforts de Weibo pour imposer la censure de la langue interdite, leur permettant ainsi de discuter de sujets controversés tels que la corruption gouvernementale.
+* **Qui était impliqué ?** [Weibo](/entities/weibo) a développé et déployé un système d'IA, qui a nui à Weibo et au [gouvernement chinois](/entities/chinese-government).
+* 🚨 **Note de l'éditeur :** La définition de "préjudice" présumé à une partie n'indique pas qu'il incombe à la communauté au sens large d'atténuer ou de prévenir ce préjudice. Bien que cet incident réponde aux critères, les éditeurs de bases de données ne prétendent pas s'il faut éviter que cet incident ne se reproduise. L'AIID indexe tous les incidents répondant à sa [définition](/research/1-criteria) ; notre responsabilité est de faire connaître de tels incidents (par exemple, [Incident #13](/cite/13), où les modèles de toxicité du langage se sont avérés être facilement trompés).
 
-**Incident #383:** [Google Home Mini Speaker Reportedly Read N-Word in Song Title Aloud](/cite/383)
+**Incident #383 :** [Le mini haut-parleur Google Home aurait lu à voix haute N-Word dans le titre de la chanson](/cite/383)
 
-* **What happened?** Google Home Mini speaker was reported by users for announcing aloud the previously-censored n-word in a song title. It is unclear when or how Google's speakers stopped censoring the n-word.
-* **Who was involved?** [Google Home](/entities/google-home) developed and deployed an AI system, which harmed [Black Google Home Mini users](/entities/black-google-home-mini-users) and [Google Home Mini users](/entities/google-home-mini-users).
+* **Que s'est-il passé ?** Le haut-parleur Google Home Mini a été signalé par des utilisateurs pour avoir annoncé à haute voix le mot n précédemment censuré dans un titre de chanson. On ne sait pas quand ni comment les haut-parleurs de Google ont cessé de censurer le n-word.
```

Comment:
Is there a reason why it's "le mot n" the first time and then "le n-word" the second time?